services/app-api/forms/mcpar.json

```diff
@@ -4379,7 +4371,7 @@
                 "validation": "number",
                 "props": {
                   "label": "D1.XIII.1 Total standard PA requests",
-                  "hint": "Enter the total number of standard PA requests logged by the plan during the prior calendar year. If you choose not to respond prior to June 2026, enter \"NR\" for not reporting."
+                  "hint": "Enter the total number of standard PA requests logged by the plan during the prior calendar year. If you choose not to respond prior to June 2026, enter “NR” for not reporting."
                 }
```

Comment:
question about this--if the BOs want the user to input 'NR', but the validation is looking for 'N/A' or 'NR', Is that ok?

Reply:
Yes, wherever users can put in "NR", "N/A" will also be a valid answer for these new sections